app/src/main/java/org/p2p/wallet/sell/ui/error/SellErrorFragment.kt

```diff
@@ -0,0 +1,63 @@
+package org.p2p.wallet.sell.ui.error
+
+import android.os.Bundle
+import android.view.View
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BaseFragment
+import org.p2p.wallet.databinding.FragmentSellErrorBinding
+import org.p2p.wallet.swap.ui.orca.OrcaSwapFragment
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.popBackStack
+import org.p2p.wallet.utils.replaceFragment
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+
+private const val ARG_ERROR_STATE = "ARG_ERROR_STATE"
+
+class SellErrorFragment : BaseFragment(R.layout.fragment_sell_error) {
```

Comment:
and again, this screen is out of scope of this task

you have https://p2pvalidator.atlassian.net/browse/PWN-6277 for corner cases